Vagrantfile

```diff
@@ -0,0 +1,24 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://atlas.hashicorp.com/search.
+  config.vm.box = "hashicorp/precise64"
+
+  # By default, running 'python app.py' will create a server for the app,
+  # which can be accessed at localhost:8080.
+  # 
+  # Therefore, We specify a guest port number that is not 80 to
+  # avoid conflicting with the Python app.
+  config.vm.network "forwarded_port", guest: 8080, host: 8080
```

Comment:
Note: when we get the latest version of Vagrant which is Vagrant 1.9.3, there is a vagrant bug which fails if we do "vagrant up". See: [https://github.com/rapid7/metasploitable3/issues/106](https://github.com/rapid7/metasploitable3/issues/106) if you want to know what the error is.

The fix can be done by adding a host_ip parameter to the line above (line 14) in this file (Vagrantfile). It should look like this instead:
`config.vm.network "forwarded_port", guest: 8080, host: 8080, host_ip: "127.0.0.1"`

Consider updating the file or leave a comment about this somewhere.